### ISSUE TITLE
Need Material Design implemented as part of Visual quite like Xamarin.Forms

### ISSUE THREAD

**Issue by egvijayanand** [dotnet/maui#9689]
### Description

Xamarin.Forms has a [NuGet](https://www.nuget.org/packages/Xamarin.Forms.Visual.Material) package for implementing the Material Design as part of Visual.

As understood from the recent [conversation](https://twitter.com/egvijayanand/status/1563198363541323776?s=20&t=G3xtUPnKGC4R5JD0rSPMJw), the Drawn control getting implemented as part of Microsoft.Maui.Graphics is being suggested as an alternative for the Visual theming moving forward.

Since there's a huge gap in the set of controls available in the native implementation of .NET MAUI and Drawn controls, it's wise to release a package quite like Xamarin.Forms to bring the Material Design for .NET MAUI App too as it gives a premium look and feel with a very minimal effort for a full-fledged production app.

### Public API Changes

Material Design as a separate NuGet package and implementation via the Visual property.

### Intended Use-Case

Material Design as part of the App look and feel.

**Comment by egvijayanand** [comment]
Drawn controls are a good initiative and I appreciate that but at the same time it's experimental and there's a huge gap in the set of controls supported for now. 

Hence it can't be a viable replacement for the Visual package of Xamarin.Forms. It can only co-exist and be available as another option.

**Comment by PureWeen** [comment]
@egvijayanand one reason we had to pull this from MAUI is that we were using the iOS Material bindings are no longer supported https://github.com/material-components/material-components-ios. 

So it didn't make sense to carry those over into MAUI unfortunately.

**Comment by egvijayanand** [comment]
I do agree that support for iOS has been made obsolete.

Are there any chances of supporting it on Android alone as it is still pretty much alive and enabling it that provides a premium look and feel to the controls?

**Comment by egvijayanand** [comment]
The reason this is being requested is that the Drawn controls are still a long way to go to match what's already supported in Xamarin.Forms and a solid alternative would be required to catch up with the migration projects.

**Comment by NPadrutt** [comment]
> Are there any chances of supporting it on Android alone as it is still pretty much alive and enabling it that provides a premium look and feel to the controls?

I wonder about this too. Currently my app looks quite outdated on Android when ported over to .net maui. Shouldn't there be a way to define a own style in the Platform - Android folder and define a new parent theme that supports Material Design?

**Comment by KieranMaclagan** [comment]
Agreed, having Material available on Android would make out-of-the-box MAUI look and feel way more polished.
